eventmesh-common/src/main/java/org/apache/eventmesh/common/protocol/tcp/codec/Codec.java

```diff
@@ -74,12 +74,20 @@ public void encode(ChannelHandlerContext ctx, Package pkg, ByteBuf out) throws E
             int headerLength = ArrayUtils.getLength(headerData);
             int bodyLength = ArrayUtils.getLength(bodyData);
 
-            int length = 4 + 4 + headerLength + bodyLength;
+            int length = headerLength + bodyLength;
```

Comment:
this line need to change:
`int length = CONSTANT_MAGIC_FLAG.length + VERSION.length + headerLength + bodyLength;`